src/groups/mqb/mqbc/mqbc_storagemanager.t.cpp

```diff
@@ -768,14 +768,14 @@ struct TestHelper {
             handle,
             uri_t,
             rec.d_queueKey,
-            mqbs::DataStore::AppIdKeyPairs(),
+            mqbs::DataStore::AppInfos(),
```

Comment:
This is okay as long as `si.amw.bmq.stats` is not a fanout domain.

Reply:
If`AppInfos` is empty, we generate storage key (as before)